client/src/components/StatusLabel/StatusLabel.tsx

```diff
@@ -54,13 +57,17 @@ const statusLabelStyling = (status: string) => {
         ? "#e9ffe8"
         : label === "rejected"
           ? "#ffeeee"
-          : "#dbe7f98f",
+          : label === "expired"
+            ? "#fff5e6"
```

Comment:
Not in this PR but it'd be nice if we can have color tokens. So it'd look something like ..
```suggestion
            ? colorTokens.expired // "#fff5e6"
```

Reply:
Seems reasonable let me implement this!